ddtrace/contrib/asyncio/__init__.py

```diff
@@ -0,0 +1,41 @@
+"""
+``asyncio`` module hosts the ``AsyncioContextProvider`` that follows the execution
+flow of ``Task``, making possible to trace asynchronous code built on top
+of ``asyncio``. To enable the provider, in your code you should::
+
+    from ddtrace import tracer
+    from ddtrace.contrib.asyncio import context_provider
+
+    # enable asyncio support
+    tracer.configure(context_provider=context_provider)
```

Comment:
@LotharSee the context provider is explicit only in this case (for the moment). Is it confusing? ``aiohttp`` uses it under the hood, and the same does ``gevent`` and (probably) ``tornado``.